src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -335,16 +332,19 @@ function get_item_html($itemkey) {
 
 	// merge data into html template, indent HTML, and return
 
-	$data = vsprintf($data_template, $args);
-	if (strlen($data) > 0) {
-		$html = "<th><span style='font-weight:normal; margin-left:20px'>{$title_content}</span></th>\n" .
-			"<td>\n{$data}\n</td>\n";
-		return str_replace("\n", "\n\t\t\t", $html);
-	} else {
-		return '';
+	if (strlen($data_template) > 0) {
```

Comment:
Make this even more robust. e.g. when there is no temperature data or... then maybe $data_template is empty and we do not want to vsprintf() at all.